app/controllers/service_controller.rb

```diff
@@ -16,6 +16,11 @@ def show_list
     process_show_list(:dbname => :service, :gtl_dbname => :service)
   end
 
+  def show
+    super
+    @view, @pages = get_view(Vm, :parent => @record, :parent_method => :all_vms, :all_pages => true)
```

Comment:
I don't understand why this is necessary?

Reply:
Before removing the explorer page, the `service_controller/show` method was calling `get_node_info` which contained the data needed to render the VM's list from [service_controller.rb#L340](https://github.com/ManageIQ/manageiq-ui-classic/blob/1615664147d606bc8e607d445cf4f06af091c452/app/controllers/service_controller.rb#L340) in the service summary's details tab. These methods were removed in a different incomplete PR and then continued the task here in [De explorization of Service Pages #8229](https://github.com/ManageIQ/manageiq-ui-classic/pull/8229/files#diff-9f30f45de34f12165b8fce7741243deeb55697dd75f465c09eaf4a3c5d07eb6a)

`@view` is used inside in [_svcs_show.html.haml#L39](https://github.com/ManageIQ/manageiq-ui-classic/blob/1615664147d606bc8e607d445cf4f06af091c452/app/views/service/_svcs_show.html.haml#L39) where the tabs are present. Vm's list are showing up empty if we do include this line here.

Please see the after screenshot **'No Tabs - VM's list'** in the description above which contains data in Vm's list.